src/content/en/2024/sustainability.md

```diff
@@ -309,7 +309,7 @@ The amount of requests, in general, is quite high, so let's see what resources t
 {{ figure_markup(
   image="number-of-requests-by-percentile-and-type-mobile.png",
   caption="Number of requests per percentile and resource type (mobile)",
-  description="TODO",
+  description="A column chart showing that on mobile devices, on the 90th percentile we find 12 requests targeting HTML content, 68.5 requests for javascript resources, almost 26 requests for CSS, around 55 requests fetching images and 8.5 requests for fonts. On the 75th percentile, there are 5 HTML requests, 41 JavaScript requests, 15 CSS requests, 29.5 images requests and 5.5 Fonts requests. On the 50th percentile, we can see 2 HTML requests, 22 javascript requests, 7.5 CSS requests, 15.5 images and 3 fonts requests. On the 25th percentile, there are 1 HTML, 10.5 javascript, 3.5 CSS, 7.5 images and 1 font requests. On the 10h percentile, we find 1 Html, 4.5 javascript, 1.5 CSS, 4 images and 0 font requests.",
```

Comment:
It's weird to have decimals in here since we are talking about number of requests but since we did not round it for the chart I will also not round it for the description

Reply:
IMHO that’s fine since it’s an average across millions of sites.